rust/lance-encoding/src/data.rs

```diff
@@ -293,6 +293,55 @@ impl FixedWidthDataBlock {
     }
 }
 
+pub struct VariableWidthDataBlockBuilder1 {
```

Comment:
for variable width data with offset u64, I will do
```
pub struct VariableWidthDataBlockBuilder2 {
    offsets: Vec<u64>,
    bytes: Vec<u8>,
}
```
it will be added in a follow up PR.

Reply:
Let's use better names please.  Even if it's just `VariableWidthDataBlockBuilder` and `LargeVariableWidthDataBlockBuilder`.

Reply:
haha, sure! `VariableWidthDataBlockBuilder` and `LargeVariableWidthDataBlockBuilder` are way better than `...1`, `...2`.

Reply:
fixed.